clients/src/test/java/org/apache/kafka/clients/ClusterConnectionStatesTest.java

```diff
@@ -40,7 +40,12 @@
     private final MockTime time = new MockTime();
     private final long reconnectBackoffMs = 10 * 1000;
     private final long reconnectBackoffMax = 60 * 1000;
-    private final double reconnectBackoffJitter = 0.2;
+    private final long connectionSetupTimeoutMs = 10 * 1000;
+    private final long connectionSetupTimeoutMaxMs = 127 * 1000;
+    private final static int RECONNECT_BACKOFF_EXP_BASE = 2;
+    private final static double RECONNECT_BACKOFF_JITTER = 0.2;
+    private final static int CONNECTION_SETUP_TIMEOUT_EXP_BASE = 2;
+    private final static double CONNECTION_SETUP_TIMEOUT_JITTER = 0.2;
```

Comment:
nit: It may be better to keep them as lower case to stay inline with the other constants. Or shall we convert all the others to upper case and to static constant?

Reply:
We can just make the constants in `ClusterConnectionStates` package-private and reuse them here?

Reply:
Make sense. I've made them package-private and lower-cased the constants.